sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/parser/AstBuilder.scala

```diff
@@ -247,6 +247,33 @@ class AstBuilder extends DataTypeAstBuilder
     WhileStatement(condition, body, Some(labelText))
   }
 
+  override def visitSearchedCaseStatement(ctx: SearchedCaseStatementContext)
```

Comment:
nit: can we do like:
```
override def visitSearchedCaseStatement(
    ctx: SearchedCaseStatementContext): SearchedCaseStatement = {
```

it's a bit cleaner, this with : in the front is a bit strange